src/components/layout/AppHeader.vue

```diff
@@ -44,16 +45,17 @@ interface Props {
 
 defineProps<Props>();
 
+const campaignParams = inject<string>( injectionKey );
```

Comment:
You could provide `''` as a default 2nd parameter, that might help with TypeScript screaming at us (or having random errors because of a null value instead of empty string